SUPPORT.md

```diff
@@ -0,0 +1,148 @@
+
+Before reporting an issue on this repository's bug tracker, please carefully read this document. There are various resources that are more likely to be able to help you with certain kinds of issues. This document provides instructions on how to determine which place best to report to.
+
+*Note: While this guide generally is more geared towards end-user problems, modders should also read it before reporting any issue with Forge in order to ensure it actually is one. This does of course not cover feature suggestions.*
+
+### Contents
+
+ 1. [Identifying the source of a problem](#identifying-the-source-of-a-problem)
+    1. [Analyzing a stack trace](#analyzing-a-stacktrace)
+    2. [Isolating an issue](#isolating-an-issue)
+ 2. [Choosing the correct support channel](#choosing-the-correct-support-channel)
+    1. [Forge support](#forge-support)
+    2. [Mod support](#mod-support)
+    3. [Vanilla support](#vanilla-support)
+ 3. [Writing a helpful bug report](#writing-a-helpful-bug-report)
+    1. [Preventing duplicates and unnecessary reports](#preventing-duplicates-and-unnecessary-reports)
+    2. [Required Information](#required-information)
+    3. [Markdown Issue Template](#markdown-issue-template)
+    4. [Waiting for an issue to be resolved](#waiting-for-an-issue-to-be-resolved)
+
+Identifying the source of a problem
+-----------------------------------
+
+You're most likely to get fast and valuable responses to your issues if you report them in the correct place. For this, you need to find the root cause of the problem.
+
+The most important source for this information are the log files Forge creates for every run of the game. You can find them in the `logs` subdirectory of your Minecraft game directory. The most important log file is `fml-[client|server]-latest.log` (depending on whether you run a singleplayer instance/connect to a server or are running a dedicated server) which contains all of Forge's and any mod's output.
+
+If you're faced by a crash, consult these logs files first. The most relevant information can be found at the very end, where you're likely to see a crash report. Since Minecraft is a Java game, crashes are expressed as so called Exceptions and information about where such an Exception occurred is listed in what is known as a stacktrace. A stacktrace simply is a hierarchy of calls that lead up to a certain point in a program.
+
+You can quickly identify a stacktrace by the line that starts it, which is of the following format:
+
+```
+Exception in thread "..." path.to.Exception
+```
+
+This is followed by a number of `at` lines, which list the call chain that lead to the Exception being thrown. The further down you go this list, the earlier a call occurred. This is the most crucial part of any crash report, since it contains information about which mod or library caused an error.
```

Comment:
To be perfectly honest, I wrote this particular section without even looking at a crash report and that at a time when I wasn't supposed to be writing anything anymore, anyways. Thanks for your suggestion, I'll just use it :D